exaudfclient/base/javacontainer/script_options/parser_ctpg_script_importer.h

```diff
@@ -16,24 +16,39 @@ namespace JavaScriptOptions {
 
 namespace CTPG {
 
+
+
 class ScriptImporter {
 
     public:
         ScriptImporter(SwigFactory & swigFactory, Keywords & keywords);
 
         void importScript(std::string & scriptCode, ExecutionGraph::OptionsLineParser::CTPG::options_map_t & options);
 
+    private:
+         struct ReplacedScripts {
+            ReplacedScripts(ReplacedScripts&&) = default;
+            std::string script;
+            size_t origPos;
+            size_t origLen;
+        };
+
     private:
         void importScript(std::string & scriptCode,
                             ExecutionGraph::OptionsLineParser::CTPG::options_map_t & options,
                             const size_t recursionDepth);
          const char* findImportScript(const std::string & scriptKey);
+
+         void replaceScripts(const ExecutionGraph::OptionsLineParser::CTPG::options_map_t::mapped_type & option_values,
```

Comment:
Adding a typedef could improve the readability of this method signature.

e.g.
```cpp
typedef ExecutionGraph::OptionsLIneParser::CTPG::options_map_t::map_type CtpgValues;

void replaceScript(const CtpgValues& options, ...)
```

Reply:
Done